### PR TITLE
add senet

### PR DESCRIPTION
**Pull request by wangxinxin08**
1. polish resnet
2. add senet

### CODE REVIEW

**Review by jerrywgz — COMMENTED** [review]
name_adapter.py could be removed?

**Comment by wangxinxin08** [comment]
> name_adapter.py could be removed?

add_sublayer里面的name是structure name，不太好remove

File: ppdet/modeling/backbones/senet.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (jerrywgz) [inline ppdet/modeling/backbones/senet.py]: 2021

File: ppdet/modeling/backbones/senet.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import paddle.nn.functional as F
+from ppdet.core.workspace import register, serializable
+from .resnet import ResNet, Blocks, BasicBlock, BottleNeck
+
+__all__ = ['SENet', 'SERes5Head']
+
+
+@register
+@serializable
+class SENet(ResNet):
```
Comment (jerrywgz) [inline ppdet/modeling/backbones/senet.py:26]: 需要添加注释

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request closed without merging**